Add documentation for using default values with triggers (#4179)
harness/developer-hub: docs/platform/triggers/triggers-reference.md

Always make sure the path you use works with the provider's payload format and the event type.

:::note

> Check [ternary operator](https://developer.harness.io/kb/continuous-delivery/articles/ternary-operator/) for instructions on how to use default values for pipeline inputs based on the trigger's payload.

Reviewer: ```suggestion
For instructions on using default values for pipeline inputs based on a trigger's payload, go to [ternary operator](https://developer.harness.io/kb/continuous-delivery/articles/ternary-operator/).
```

You can use [built-in Git payload expressions](#built-in-git-payload-expressions) and [JEXL expressions](#jexl-conditions) in this setting.

:::note

> Check [ternary operator](https://developer.harness.io/kb/continuous-delivery/articles/ternary-operator/) for instructions on how to use default values for pipeline inputs.

Reviewer: ```suggestion
For instructions on using default values in pipeline inputs, go to [ternary operator](https://developer.harness.io/kb/continuous-delivery/articles/ternary-operator/).
```
